tools/clang/unittests/HLSL/ShaderOpTest.cpp

```diff
@@ -9,6 +9,9 @@
 //                                                                           //
 ///////////////////////////////////////////////////////////////////////////////
 
+// We need to keep & fix these warnings to integrate smoothly with HLK
+#pragma warning(error: 4100 4146 4242 4244 4267 4701 4389)
+
```

Comment:
It looks like some code I wrote is causing the failure here. I recommend changinge line 300 to: `CD3DX12_GPU_DESCRIPTOR_HANDLE gpuHandle = CD3DX12_GPU_DESCRIPTOR_HANDLE();`

Reply:
Yes, but I also did a wrong rebase. I'll push a new update.